WordPress/src/main/java/org/wordpress/android/widgets/WPNetworkImageView.java

```diff
@@ -278,6 +303,33 @@ private void handleResponse(ImageLoader.ImageContainer response, boolean isCache
         }
     }
 
+    /*
+     * crops the passed bitmap to fit the passed width/height - bitmap will be resized if
+     * necessary to ensure it fills the desired size before cropping
+     */
+    private static Bitmap cropBitmap(Bitmap bitmap, int newWidth, int newHeight) {
```

Comment:
There is a `ThumbnailUtils` helper in Android, that's supposed to handle cropping in situations like this, so maybe we could replace call to this method with `ThumbnailUtils.extractThumbnail(bitmap, mCropWidth, mCropHeight);` ?

They produce really similar results:
![crop_comp](https://cloud.githubusercontent.com/assets/728822/19502949/3e0132de-95eb-11e6-9332-bbd0f274d608.jpg)